src/test-func/groovy/org/transmartproject/batch/highdim/mrna/data/MrnaMultipleSamplesTests.groovy

```diff
@@ -0,0 +1,74 @@
+package org.transmartproject.batch.highdim.mrna.data
+
+import org.junit.AfterClass
+import org.junit.ClassRule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.rules.TestRule
+import org.junit.runner.RunWith
+import org.springframework.context.annotation.AnnotationConfigApplicationContext
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner
+import org.transmartproject.batch.beans.GenericFunctionalTestConfiguration
+import org.transmartproject.batch.clinical.db.objects.Tables
+import org.transmartproject.batch.db.TableTruncator
+import org.transmartproject.batch.junit.JobRunningTestTrait
+import org.transmartproject.batch.junit.RunJobRule
+import org.transmartproject.batch.support.TableLists
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.*
+
+/**
+ * test mRNA data import in the simplest scenario (good data not previously
+ * loaded)
+ */
+@RunWith(SpringJUnit4ClassRunner)
+@ContextConfiguration(classes = GenericFunctionalTestConfiguration)
+class MrnaMultipleSamplesTests implements JobRunningTestTrait {
+
+    private final static String STUDY_ID = 'MULTSAMPL'
+    private final static String PLATFORM_ID = 'GPL570_bogus'
+
+    @Test
+    void testMultipleSamplesMeasurements() {
+        def params = [study_id: STUDY_ID]
+
+        List r = jdbcTemplate.queryForList """
+                SELECT DISTINCT S.sample_cd
+                FROM
+                    ${Tables.MRNA_DATA} D
+                    INNER JOIN ${Tables.SUBJ_SAMPLE_MAP} S ON (D.assay_id = S.assay_id)
+                WHERE D.trial_name = :study_id""",
+                params
+
+        assertThat r, containsInAnyOrder(
+                hasEntry('sample_cd', 'S11'),
+                hasEntry('sample_cd', 'S12'),
+                hasEntry('sample_cd', 'S21'),
+                hasEntry('sample_cd', 'S22'),
```

Comment:
Testing that the samples are associated with the correct patient would be interesting too.